credential-trust-establishment/spec.md

```diff
@@ -215,10 +222,70 @@ Linked governance files provide a flexible mechanism for governing larger ecosys
 
 Linked governance allows for governance chains to exist - the inclusion of an external governance file allows that governance to itself reference other governance files. While in theory this allows for infiniately long chains, in practice the requirement for the publisher to trust linked files places a reasonable limit. Also see (Expectations)[#expectations] for further commentary on the responsiblities of the publisher and reader.
 
+### Versioning
+
+Versioning is a way to allow users of decentralized ecosystem governance to track/use the current version of published governance or track/use older version of the file. This allowes flexibility of using/tracking most current version of DEGov file at any time.
+
+The version of the current file is indicated by an integer. Each subsequent published version of this file must increment this version number by 1, ensuring a clear and linear version history. Example:
+
+```json
+{
+  "version": 1
+}
+```
+
+The version (stringified) is used to generate a unique name of the current version of governance file followed by the file extension (.json) Example:
```

Comment:
can we leave off the requirement to derive the filename from the version? What do we lose if we do that?